backend/app/views/spree/admin/return_authorizations/_form.html.erb

```diff
@@ -9,13 +9,13 @@
             <%= check_box_tag 'select-all' %>
           <% end %>
         </th>
-        <th class="return-item-product"><%= Spree.t(:product) %></th>
-        <th class="return-item-state"><%= Spree.t(:state) %></th>
-        <th class="return-item-charged"><%= Spree.t(:charged) %></th>
-        <th class="return-item-pre-tax-refund-amount"><%= Spree.t(:pre_tax_refund_amount) %></th>
-        <th class="return-item-reimbursement-type"><%= Spree.t(:reimbursement_type) %></th>
-        <th class="return-item-exchange-for"><%= Spree.t(:exchange_for) %></th>
-        <th class="return-item-reason"><%= Spree.t(:reason) %></th>
+        <th class="return-item-product"><%= Spree::Product.model_name.human %></th>
+        <th class="return-item-state"><%= Spree::ReturnAuthorization.human_attribute_name(:state) %></th>
+        <th class="return-item-charged"><%= Spree::ReturnItem.human_attribute_name(:charged) %></th>
+        <th class="return-item-pre-tax-refund-amount"><%= Spree::LineItem.human_attribute_name(:pre_tax_amount) %></th>
+        <th class="return-item-reimbursement-type"><%= Spree::ReimbursementType.model_name.human %></th>
```

Comment:
This is the label for a input field for a `ReturnItem`, so we should not use the `LineItem`s attributes here. 

Besides, the actual column has been renamed, so the key should be named `amount`. The translation in english can still be `Amount before sales tax`.